src/components/FormValidator.js

```diff
@@ -16,7 +16,6 @@ class FormValidator{
     }
     
     _hideInputError(inputElement){
-
       const errorMessageElement = this._form.querySelector(`#${inputElement.id}_error`)
       inputElement.classList.remove(this._inputErrorClass);
       errorMessageElement.textContent = " ";
```

Comment:
Please, delete lines 46-47  from `resetValidation`. You have `this.toggleButtonState();` which can disable buttons if the inputs are empty

```JS

    resetValidation() {
      this.toggleButtonState();
      this._inputElements.forEach((inputElement) => {
        this._hideInputError(inputElement)
      });
    }
```